iceoryx_hoofs/source/log/building_blocks/console_logger.cpp

```diff
@@ -100,30 +107,34 @@ void ConsoleLogger::createLogMessageHeader(const char* file,
         strncpy(&timestampString[0], &TIME_FORMAT[0], ConsoleLogger::bufferSize(TIME_FORMAT));
     }
 
-    constexpr auto MILLISECS_PER_SECOND{1000};
-    auto milliseconds = static_cast<int32_t>(timestamp.tv_nsec % MILLISECS_PER_SECOND);
+    constexpr uint32_t MILLISECS_PER_SECOND{1000};
+    const auto milliseconds = static_cast<int32_t>(timestamp.tv_nsec % MILLISECS_PER_SECOND);
 
     /// @todo iox-#1345 do we also want to always log the iceoryx version and commit sha? Maybe do that only in
-    /// `initLogger` with LogDebug
+    /// 'initLogger' with LogDebug
 
     /// @todo iox-#1345 add an option to also print file, line and function
     unused(file);
     unused(line);
     unused(function);
 
+    // AXIVION Next Construct AutosarC++19_03-A3.9.1 : Not used as an integer but as string literal
+    // AXIVION DISABLE STYLE AutosarC++19_03-M2.13.2 : Required for the color codes; only valid octal digits are used
     constexpr const char* COLOR_GRAY{"\033[0;90m"};
+    // AXIVION Next Construct AutosarC++19_03-A3.9.1 : Not used as an integer but as string literal
+    // AXIVION DISABLE STYLE AutosarC++19_03-M2.13.2 : Required for the color codes; only valid octal digits are used
     constexpr const char* COLOR_RESET{"\033[m"};
```

Comment:
Did you mean?
```suggestion
    constexpr const char* COLOR_RESET{"\033[m"};
    // AXIVION ENABLE STYLE AutosarC++19_03-M2.13.2
```

Reply:
:grimacing: good catch. Was an c&p error and it should have been `AXIVION Next Construct AutosarC++19_03-M2.13.2`

There is also an issue in `function_ref.hpp` with `// AXIVION DISABLE STYLE AutosarC++19_03-A16.2.3` but it is not enabled again. I think this can completely be removed since the comment is wrong.

Reply:
@elBoberido AFAIK `DISABLE` is only active in one file.